mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -92,21 +108,23 @@ CLASS ail net/minecraft/entity/Entity
 		ARG 1 cursorDeltaX
 		ARG 3 cursorDeltaY
 	METHOD a requestTeleport (DDD)V
-		ARG 1 x
-		ARG 3 y
-		ARG 5 z
+		ARG 1 destX
+		ARG 3 destY
+		ARG 5 destZ
 	METHOD a setPositionAnglesAndUpdate (DDDFF)V
```

Comment:
This isn't necessary. `requestTeleport(x, y, z)` means "request teleport to x, y, z". Look at our her methods such as Entity.move which have parameters also names just "x, y, z".

Reply:
I don't see any such example method. Besides, "x, y, z" can have other interpretations. They can be an offset on each axis, a range within which to teleport, or width/height/depth. In this case it's very clearly the _destination_ you want to teleport to, so I see value in making that distinction with the names.

Reply:
It's clear they're absolute positions. Every single other method that sets positions (and there are many) uses `x`, `y`, `z`. 

> I don't see any such example method. 

Open the "Entity" class, do a search for "move(".

Reply:
There is only one method here I see named `move`

    public void move(MovementType type, Vec3d offset) {

Reply:
Yeah, Mojang moved everything over to Vec3d recently, which uses `x`, `y`, and `z` as it's general-purpose. I'm okay with both `x,y,z` and `destX,destY,destZ` here.